rpc_jobs/setup_statsd.yml

```diff
@@ -0,0 +1,129 @@
+# Merge Trigger job for setup statsd.
+# This used to redeploy whenever there are merges to rpc-gating.
+- job:
+    name: "Merge-Trigger-StatsD"
+    project-type: freestyle
+    scm:
+      - git:
+          url: https://github.com/rcbops/rpc-gating
+          branches:
+           - master
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    triggers:
+        - github
+    builders:
+      - trigger-builds:
+        - project:
+            - "Setup-StatsD"
```

Comment:
Cleaner to have a separate merge trigger job for both statsd and graphite, or better to combine them into one?

Reply:
I've registered a follow up card to do the deduplication: https://rpc-openstack.atlassian.net/browse/RE-1472